data/manifest_core.json

```
@@ -5,55 +5,199 @@
         "subtype": {
           "h5n1": {
             "segment": {
-              "ha": "",
-              "mp": "",
-              "na": "",
-              "ns": "",
-              "np": "",
-              "pa": "",
-              "pb1": "",
-              "pb2": "",
+              "ha": {
+                "resolution": {
+                  "all-time": "",
+                  "2y": "",
+                  "default": "2y"
+                }
```

Bunch of missing closing braces, e.g.

```suggestion
                }
              },
```

here and in other changed places.

---

I fixed and [repushed](https://github.com/nextstrain/nextstrain.org/compare/fc223a7e3cf8d8a6cf4a66c1eb53a4a079a1b80a..dfb2af56d8ffbc00faecf909b67cb43ffaa97774) so we can preview.